src/fragments/lib/auth/common/totp/flows.mdx

```diff
@@ -0,0 +1,89 @@
+
+## Setup
+
+<Callout warning>
+
+The following options are only available when starting a new project (via `amplify add auth`). You will not have access to these settings after creation (via `amplify update`).
+
+- Required MFA
+
+</Callout>
+
+<BlockSwitcher>
+
+<Block name="New Project">
+
+Run `amplify add auth` to create a new Cognito Auth resource, and follow the prompts below to set up TOTP in your flow.
+
+
+### As a verification method
+
+import all1 from "/src/fragments/lib/auth/common/totp/add_verification.mdx";
+
+<Fragments fragments={{all: all1}} />
+
+### TOTP MFA
+
+import all2 from "/src/fragments/lib/auth/common/totp/add_mfa.mdx";
+
+<Fragments fragments={{all: all2}} />
+
+</Block>
+
+<Block name="Existing Auth">
+
+Run `amplify update auth` and follow the prompts as guided below.
+
+### As a verification method
+
+import all4 from "/src/fragments/lib/auth/common/totp/update_verification.mdx";
+
+<Fragments fragments={{all: all4}} />
+
+### TOTP MFA
+
+import all5 from "/src/fragments/lib/auth/common/totp/update_mfa.mdx";
+
+<Fragments fragments={{all: all5}} />
+
+</Block>
+
+</BlockSwitcher>
+
+## Sign In
+
+When MFA is marked as Required or **ON** for the user, and TOTP has been not been set up, a `signIn` call will return `continueSignInWithTOTPSetup` as the next step. This step signifies that the user must set up TOTP before they can sign in. The step returns an associated value of type `TOTPSetupDetails` that contains the `sharedSecret` that will be used to either to generate a QR code or can be manually entered into an Authenticator app. Once the authenticator app is set up, the user can generate a TOTP code and provide it to the library to complete the sign in process.
```

Comment:
Small tweaks:

When MFA is marked as Required or **ON** for a user, and TOTP has been not been set up for their account, a `signIn` call will return `continueSignInWithTOTPSetup` as the next step. This step signifies that the user must set up TOTP before they can sign in. The step returns an associated value of type `TOTPSetupDetails` that contains the `sharedSecret` that will be used to either to generate a QR code or can be manually entered into an Authenticator app. Once the authenticator app is set up, the user can generate a TOTP code and provide it to the library to complete the sign in process.

Reply:
Thanks. I guess I mixed up certain things. Updated to clarify some details.